.github/workflows/test_terraform_files.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: Terraform files test
+
+on:
+  pull_request:
+    paths:
+      - 'terraform/**'
+
+permissions:
+  contents: read
+
+jobs:
+  validate:
```

Comment:
Do you think it makes sense to add this jobs [here](https://github.com/canonical/operator-workflows/blob/main/.github/workflows/test.yaml) instead?